src/main/java/uk/gov/register/db/EntryQueryDAO.java

```diff
@@ -18,7 +18,7 @@
 public interface EntryQueryDAO {
     @RegisterMapper(EntryMapper.class)
     @SingleValueResult(Entry.class)
-    @SqlQuery("select entry_number, sha256hex, timestamp, key from entry where entry_number=:entryNumber")
+    @SqlQuery("select e.entry_number, array_agg(ei.sha256hex) as sha256hex, e.timestamp, e.key from entry e join entry_item ei on ei.entry_number = e.entry_number where e.entry_number = :entryNumber group by e.entry_number, e.timestamp, e.key")
```

Comment:
I also don't think you need `e.timestamp` and `e.key` in the `group by`, given that it is already grouping by `e.entry_number` and this is the primary key of the `entry` table.